tools/build/build.js

```diff
@@ -60,7 +60,14 @@ const taskDm = new Task('dm')
     await exec(compiler, 'tgstation.dme');
   });
 
-runTasks([
+// Frontend
+let tasksToRun = [
   taskTgui,
   taskDm,
-]);
+];
+
+if (process.env['TG_BUILD_TGS_MODE']) {
+  tasksToRun.pop();
```

Comment:
You don't need to do this, rask runner runs whatever you supply in arguments, so just passing "tgui" or "dm" would suffice.

Reply:
this seemed like a good way to have "everything EXCEPT dm"